pkg/kubelet/status/manager.go

```diff
@@ -127,6 +127,22 @@ func (m *manager) SetPodStatus(pod *api.Pod, status api.PodStatus) {
 		status.StartTime = oldStatus.StartTime
 	}
 
+	// Set ReadyCondition.LastTransitionTime.
+	// Note we cannot do this while generating the status since we do not have oldStatus
+	// at that time for mirror pods.
+	readyCondition := api.GetPodReadyCondition(status)
```

Comment:
nit: `if readyCondition := api.GetPodReadyCondition(status); readyCondition != nil {` would be shorter and keep the variable scope limited.